### DISCUSSION TITLE
Problems upgrading

### DISCUSSION THREAD

**Discussion by AlexanderWright** [docker-mailserver/docker-mailserver#3113] [Q&A] [answered]
My system has sent the email telling me about an upgrade:

> Current version: 11.3.0
> Latest  version: 11.3.1

But there is no change after 'docker-compose pull' and a restart. The Pull does not appear to download anything, as the command returns almost immediately. What am I doing wrong?

**Comment by polarathene** [comment, marked as answer]
You're not doing anything wrong.

There was a mistake that was intended to be resolved with a new version being published before the end of January, but due to a considerable amount of activity going into the next release (v12), this has dragged out to being planned for March.

You can find more details on the pinned issue here: https://github.com/docker-mailserver/docker-mailserver/issues/2952
